librepo/downloader.c

```diff
@@ -566,7 +567,8 @@ lr_headercb(void *ptr, size_t size, size_t nmemb, void *userdata)
                 lrtarget->headercb_state = LR_HCS_INTERRUPTED;
                 lrtarget->headercb_interrupt_reason = g_strdup_printf(
                     "Server reports Content-Length: %"G_GINT64_FORMAT" but "
-                    "expected size is: %"G_GINT64_FORMAT,
+                    "expected size is: %"G_GINT64_FORMAT
+                    " (Most likely there is a mismatch on the server between repodata and the rpm package)",
```

Comment:
Thanks for updating the message :slightly_smiling_face: How about rephrasing it altogether, what do you think about this?

> Inconsistent server data, reported file Content-Length: %, repository metadata states file length: % (please report to repository maintainer)

Reply:
Sure, updated. :+1: